ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/apitesting/TestUtils.scala

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.apitesting
+
+import java.util.UUID
+
+import com.digitalasset.daml.lf.archive.UniversalArchiveReader
+import com.digitalasset.ledger.api.v1.command_service.SubmitAndWaitRequest
+import com.digitalasset.ledger.api.v1.command_submission_service.SubmitRequest
+import com.digitalasset.ledger.api.v1.commands.Command.Command.Create
+import com.digitalasset.ledger.api.v1.commands.{Command, Commands, CreateCommand}
+import com.digitalasset.ledger.api.v1.ledger_offset.LedgerOffset
+import com.digitalasset.ledger.api.v1.ledger_offset.LedgerOffset.LedgerBoundary.{
+  LEDGER_BEGIN,
+  LEDGER_END
+}
+import com.digitalasset.ledger.api.v1.ledger_offset.LedgerOffset.Value.Boundary
+import com.digitalasset.ledger.api.v1.transaction_filter.{
+  Filters,
+  InclusiveFilters,
+  TransactionFilter
+}
+import com.digitalasset.ledger.api.v1.value.Value.Sum.Party
+import com.digitalasset.ledger.api.v1.value.{Identifier, Record, RecordField, Value}
+import com.digitalasset.platform.PlatformApplications.Config
+import com.google.protobuf.timestamp.Timestamp
+import scalaz.syntax.tag._
+
+object TransactionFilters {
+  def allForParty(party: String) = TransactionFilter(Map(party -> Filters()))
+
+  def onlyTemplatesByParty(templatesByParty: Map[String, Seq[Identifier]]) =
+    TransactionFilter(
+      templatesByParty.mapValues(templateIds => Filters(Some(InclusiveFilters(templateIds)))))
+}
+
+object LedgerOffsets {
+  val LedgerBegin = LedgerOffset(Boundary(LEDGER_BEGIN))
+  val LedgerEnd = LedgerOffset(Boundary(LEDGER_END))
+}
+
+object TestParties {
+  val Alice = "Alice"
+  val Bob = "Bob"
+}
+
+object TestUtils {
+
+  private lazy val ParsedPackageId: String =
+    UniversalArchiveReader().readFile(Config.defaultDarFile).get.main._1 // Test.dar
```

Comment:
This wont work in the tool, since the test tool passes a dar file at a different than the default location.

Reply:
yes, but this is identical to the state what we have now. I wanted to centralise this logic first and foremost. We can think about how to make it dynamic later.

Reply:
This will break Ledger API Test Tool, which is our publicly exposed tool.